### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ModischFabrications**
add windows venv path to installation guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This is useful for running specific tests.  The easiest way to set this up
 is to run:
 
 1. `tox --devenv venv`  (note: requires tox>=3.13)
-2. `. venv/bin/activate`
+2. `. venv/bin/activate` (POSIX) or `venv\Scripts\activate` (Windows)
```
Comment (asottile) [inline CONTRIBUTING.md:20]: specifically in `cmd.exe` -- if you're using `git bash` the former works

Comment (ModischFabrications) [reply CONTRIBUTING.md:20]: alright,  I was not aware that the generation from 2 differs by shell, not OS. Might want to add a sentence somewhere that everything should be executed with bash instead then?

Comment (asottile) [reply CONTRIBUTING.md:20]: I mean this patch is fine, I'm just suggesting it becomes a little bit more specific

the directions were written before most of the execution worked well on windows so it doesn't surprise me that there's parts that are missing

Comment (ModischFabrications) [reply CONTRIBUTING.md:20]: I'm back, sorry for the wait. I copied the definition from [the python documentation](https://docs.python.org/3/library/venv.html) , which won't specify it any further than "Windows". Powershell will act the same as cmd.exe, ksh & more like bash. Listing interpreters will probably have more edge cases than just calling git bash a POSIX-alike.

Comment (asottile) [reply CONTRIBUTING.md:20]: welllll we're using `virtualenv` and not `venv` so those docs don't precisely apply -- a better solution might be to do:

```markdown
[activate your virtualenv](link to docs...)
```

Comment (ModischFabrications) [reply CONTRIBUTING.md]: I'm not entirely convinced it's easier that way, but it's not worth fighting over. I changed it to your suggestions, feel free to merge if everything is alright.

**Pull request merged**